busted/outputHandlers/TAP.lua

```diff
@@ -1,5 +1,14 @@
 local pretty = require 'pl.pretty'
 local io = io
+local type = type
+local string_format = string.format
+local string_gsub = string.gsub
+local io_write = io.write
+local io_flush = io.flush
+
+local function print(msg)
```

Comment:
It's kind of confusing that methods are cached one way and used with local names (`string.format()`→`string_format()`) while global functions are not (`print()`→`print()`). Maybe this should be prefixed as a local function `_print()` and used that way throughout? (also `_type` etc.)

Reply:
If you don't think this is a good idea I'll be fine with that too. It wouldn't be my coding style preference but it gets the job done. The rest of the PR looks okay so if you decide to dismiss this rename suggestion I'll just approve instead.

Reply:
I get the point but never seen anything prefixing them with _ ? so I'd rather dismiss this one indeed.

Reply:
I see them all the time … but maybe it's mostly in my own code. And it might be a hold-over from Haskell and Rust where it is a compiler-enforced naming scheme for shadowed scope or unused values. Also note `luacheck` also processes `_` as a way to discard a value so there is some precedent for scope delineation. And since you can't use primes like in Haskell...

In any case if you don't like it no biggy.

Reply:
the single "_" to ignore something is a very common Lua pattern